### PR TITLE
6.1 start page servlet impl

### PR DESCRIPTION
**Pull request by merceneryinbox**

### CODE REVIEW

File: src/main/java/servlets/CartServlet.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			throws ServletException, IOException {
+		req.getRequestDispatcher("cart.jsp")
+				.forward(req, resp);
+	}
+	
+	@Override protected void doDelete(HttpServletRequest req, HttpServletResponse resp)
+			throws ServletException, IOException {
+		req.getRequestDispatcher("cart.jsp")
+				.forward(req, resp);
+	}
```
Comment (MrToady) [inline src/main/java/servlets/CartServlet.java]: I think we dont need put() and delete() methods right now

File: src/main/java/servlets/CartServlet.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		}
+	}
+	
+	@Override protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+			throws ServletException, IOException {
+		req.getRequestDispatcher("cart.jsp")
+				.forward(req, resp);
+	}
+	
+	@Override protected void doPut(HttpServletRequest req, HttpServletResponse resp)
```
Comment (MrToady) [inline src/main/java/servlets/CartServlet.java:28]: I think we don't need put and delete methods

File: src/main/java/servlets/RegistrationServlet.java [hunk 3]
```diff
@@ -5,13 +5,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import java.io.IOException;
 
 @WebServlet("/sign_up")
 public class RegistrationServlet extends HttpServlet {
+	String userName = null;
```
Comment (MrToady) [inline src/main/java/servlets/RegistrationServlet.java]: can be private. You don't need to assign null to String. It is already done

**Comment by MrToady** [comment]
A can not see other files from dev. Did you merged dev before request?

File: src/main/java/servlets/CartServlet.java (hunk 2 above)
Comment (acorned) [reply src/main/java/servlets/CartServlet.java:28]: at least now)

**Review by acorned — APPROVED** [review]
Agree with Taras, nothing to add.

**Pull request merged**